bindings/stronghold-nodejs/README.md

```diff
@@ -0,0 +1,44 @@
+# IOTA Identity Stronghold Node.js
+
+> This is the beta version of the official Stronghold Node.js plugin for [IOTA Identity Wasm](../wasm).
+
+## Install the plugin:
+
+Latest Release: this version matches the main branch of this repository and is stable.
+```bash
+npm install @iota/identity-stronghold-nodejs
+```
+
+Development Release: this version matches the dev branch of this repository, may see frequent breaking changes and has the latest code changes.
+```bash
+npm install @iota/identity-stronghold-nodejs@dev
+```
+
+## Build
+
+Alternatively, you can build the bindings if you have Rust installed. If not, refer to [rustup.rs](https://rustup.rs) for the installation. Then install the necessary dependencies using:
+```bash
+npm install
+```
+
+To use the latest local changes from the Wasm bindings follow the build instructions from the [IOTA Identity Wasm build section](../wasm#build) and then link the result with:
```

Comment:
I think the links should be absolute to work in the Wiki. I'm not sure if the api reference is already configured for this project or not.

Reply:
Good point, since it is deeplink we would need decide on the branch we want to link to (`main` or `dev`), since a link would look like https://github.com/iotaledger/identity.rs/blob/dev/bindings/wasm/README.md

Reply:
We currently link to `main`. Note that I believe we're still planning on removing either `dev` or `main` in the future.

Reply:
used a an absolute link to `main`.